src/main/java/racingcar/model/TryNumber.java

```diff
@@ -0,0 +1,32 @@
+package racingcar.model;
+
+public class TryNumber {
+
+    private String clearTryNumber;
+
+    public void setClearTryNumber(String tryNumber) {
+        isNumeric(tryNumber);
+        isDigits(tryNumber);
+        this.clearTryNumber = tryNumber;
+    }
+
+    public int getTryNumber() {
+        return Integer.parseInt(this.clearTryNumber);
+    }
+
+    public void isNumeric(String tryNumber) throws IllegalArgumentException {
+        for (int i = 0; i < tryNumber.length(); i++) {
+            if (!Character.isDigit(tryNumber.charAt(i))) {
+                throw new IllegalArgumentException("1이상의 정수를 입력해주세요.");
+            }
+        }
+    }
```

Comment:
```suggestion
    public void isNumeric(String tryNumber) throws IllegalArgumentException {
        try {
            Integer.parseInt(tryNumber);
        } (NumberFormatException exception){
            throw new IllegalArgumentException("1이상의 정수를 입력해주세요.");
        }   
    }
```

이렇게도 작성해 볼 수 있을 것 같아요 😄